doc/protocols/scion-header.rst

```diff
@@ -732,3 +731,255 @@ packet is dropped.
+   - A SCION path type header with slightly modified hop fields.
+
+::
+
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                             PktID                             |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                     OrigPayloadLen + Flags                    |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                          PathMetaHdr                          |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           InfoField                           |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              ...                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           InfoField                           |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           HopField                            |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                              ...                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                           HopField                            |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+SCION Header Modifications
+--------------------------
+EPIC-SAPV contains the SCION path type header with the following
+adaptation:
+
+   - The size of the MAC (six bytes for the SCION path type) inside the
+     Hop Fields is reduced to two bytes, the four bytes of freed
+     space are used for the Hop Validation Field (HVF).
+
+OrigPayloadLen + Flags
+^^^^^^^^^^^^^^^^^^^^^^
+
+::
+
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |R|V|        Reserved           |         OrigPayloadLen        |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+EPIC Path Reversal (R):
+  Indicates that the path was reversed. This information tells the
+  border routers on the return path that the HVF in the packet was
+  already updated and that the source and destination in the address
+  header were switched. This is necessary because the destination
+  address (instead of the source address) and the OrigPayloadLen
+  (instead of the PayloadLen) have to be used for the correct
+  verification of the HVF on the return path.
+
+Path Validation for the Source Host (V):
+  Specifies whether the source host wants to validate the path.
+  Setting this flag will cause the destination host to send back a
+  proof that the packet indeed traversed the path intended by the source.
+
+OrigPayloadLen:
+  For a packet traversing the path in the backward direction (``R=1``),
```

Comment:
We should probably specify what SHOULD / MUST happen in forward direction as well